### ISSUE TITLE
Bar chart crashing for @nativescript/android 8.5.1

### ISSUE THREAD

**Issue by nc1401** [nativescript-community/ui-chart#55]
**Summary**
After upgrading @nativescript/android  to 8.5.1, adding a bar chart just crashes the android app, works okay for ios and on android with version 8.4.0 of @nativescript/android

**Expected Behavior**
Bar chart should appear with no app crash

**Device (please complete the following information):**
 - Device: Google Pixel
 - Android Version - API 33
 - Library Version ^1.2.26

**Additional Context**
Tried updating @nativescript-community/ui-chart to 1.2.30, the crash is still observed.

ADD A REWARD using [Speed](speedoss.com) to SOLVE this issue QUICKLY and SUPPORT this project.

**Comment by farfromrefug** [comment]
@nc1401 the issue is uon the runtime side. It will fixed shortly

**Comment by nc1401** [comment]
@farfromrefug  Any tentative date, so we can plan our work. And should I follow tis thread for update, is that it?

**Comment by farfromrefug** [comment]
@nc1401 you can ask on discord. In the meantime.stick to an older runtime version

**Comment by nc1401** [comment]
@farfromrefug we can't get back to an older android version at the moment, cause 8.5 supports ermbedded and we need that

**Comment by farfromrefug** [comment]
@nc1401 then raise your issue on discord @edusperoni might be able to answer you about a release date

**Comment by edusperoni** [comment]
Does this issue happen on 8.5.0 or 8.5.2? Might need some additional information to look into this.

**Comment by nc1401** [comment]
@edusperoni  Not getting issue with 8.5.0, but I get issue with 8.5.2. I don't have crash logs, it just crashes, and this is a warning I see, if this helps: 

WARNING in ./node_modules/@nativescript-community/ui-chart/components/AxisBase.js 237:39-54
export 'parseDashEffect' (imported as 'parseDashEffect') was not found in '@nativescript-community/ui-canvas' (possible exports: Align, Arc, BitmapShader, Canvas, CanvasBase, CanvasView, Cap, Circle, ColorMatrixColorFilter, DEFAULT_SCALE, DashPathEffect, Direction, FillType, FontMetrics, Join, LayoutAlignment, Line, LinearGradient, Matrix, Op, Paint, Path, PathEffect, PorterDuffMode, PorterDuffXfermode, RadialGradient, Rect, RectF, Rectangle, Shape, StaticLayout, Style, Text, TileMode, UICustomCanvasView, cachedProperty, callDrawBeforeShapesProperty, createImage, createRect, createRectF, densityProperty, hardwareAcceleratedProperty, releaseImage)
 @ ./node_modules/@nativescript-community/ui-chart/components/YAxis.js 1:0-38 13:27-35
 @ ./node_modules/@nativescript-community/ui-chart/charts/BarChart.js 2:0-53 27:56-75 27:97-116 30:57-77 30:99-119
 @ ./node_modules/@nativescript-community/ui-chart/charts/index.js 1:0-38 1:0-38
 @ ../../../libs/nativescript/shared/src/lib/components/charts/charts.module.ts 4:0-88 5:0-59 7:0-7 8:34-42 9:34-42 10:35-44
 @ ../../../libs/nativescript/shared/src/index.ts 4:0-54 4:0-54
 @ ./src/main.ts 7:0-62 11:0-23

**Comment by edusperoni** [comment]
<img width="385" alt="image" src="https://github.com/nativescript-community/ui-chart/assets/4061767/0b80d17e-9834-451e-9bc2-5b435219369e">
Unless you have a way to reproduce it, I won't be able to help. Just tested on 8.5.2 and my bar chart works fine

**Comment by nc1401** [comment]
@edusperoni have you identified the issue?

**Comment by edusperoni** [comment]
Unfortunately I didn't get the time to find a reproduction. If you can provide an example I can look at it and fix it

**Comment by nc1401** [comment]
does @farfromrefug  comment not help you at all?

**Comment by farfromrefug** [comment]
@nc1401 yes it helps but to fix it (or test the fix) he needs to reproduce it. If you can create a simple reproducible example project @edusperoni cab quickly test

**Comment by nc1401** [comment]
[Files for barchart.zip](https://github.com/nativescript-community/ui-chart/files/13045591/Files.for.barchart.zip)
@farfromrefug @edusperoni  These are the relevant files, files under Barchart Component create the chart, and files under LibraryThatusesChart are calling the chart component from BarChartComponet

**Comment by edusperoni** [comment]
@nc1401 this issue should have been fixed already on latest 8.5 and 8.6

**Comment by vmutafov** [comment]
@edusperoni I was just writing this, thanks man!
